doc/source/multi_run_plans.rst

```diff
@@ -0,0 +1,511 @@
+
+    def sample_plan():
+        ...
+        bps.open_run(md={})  # 'md' - metadata to be added to the 'start' document
+        ...
+        < code that controls execution of the scan >
+        ...
+        bps.close_run()
+
+    RE(sample_plan())
+
+or using `@bpp.run_decorator`, which inserts `open_run` and `close_run` control messages
+before and after the sequnce generated by the enclosed code:
+
+.. code-block:: python
+
+    # Using 'bpp.run_decorator' to define a scan
+
+    import bluesky.preprocessors as bpp
+    from bluesky import RunEngine
+
+    RE = RunEngine({})
+
+    @bpp.run_decorator(md={})  # 'md' - metadata to be added to the 'start' document
+    def sample_plan():
+        ...
+        < code that controls execution of the scan >
+        ...
+
+    RE(sample_plan())
+
+The rules for basic Bluesky plans require that the currently running scan must be closed before
+the next scan is opened, therefore the following code works:
+
+.. code-block:: python
+
+    # This code works, since the first run is closed before the second one is opened
+
+    import bluesky.plan_stubs as bps
+    from bluesky import RunEngine
+
+    RE = RunEngine({})
+
+    def sample_plan():
+        bps.open_run(md={})
+        < code that controls execution of the scan >
+        bps.close_run()  # Closing the first run (scan)
+        bps.open_run(md={})  # Opening the second run (scan)
+        < code that controls execution of the scan >
+        bps.close_run()
+
+    RE(sample_plan())
+
+but the following code fails:
+
+.. code-block:: python
+
+    # This code fails, since the second run is opened before the first run is closed
+
+    import bluesky.plan_stubs as bps
+    from bluesky import RunEngine
+
+    RE = RunEngine({})
+
+    def sample_plan():
+        bps.open_run(md={})  # Starting the first run
+        < code that controls execution of the scan >
+        bps.open_run(md={})  # Starting the second scan before the first one is closed
+        < code that controls execution of the scan >
+        bps.close_run()
+        bps.close_run()
+
+    RE(sample_plan())
+
+
+Note, that the preassembled plans, such as `bluesky.plans.count` or `bluesky.plans.list_scan`,
+are complete single-run plans, enclosed in `open_run` and `close_run` messages, therefore
+the following code fails as well:
+
+.. code-block:: python
+
+    # This code fails while attempting to start a preassembled plan from an open run
+
+    import bluesky.plan_stubs as bps
+    from bluesky.plans import count
+    from bluesky import RunEngine
+
+    RE = RunEngine({})
+
+    def sample_plan():
+        bps.open_run(md={})  # Starting the first run
+        < code that controls execution of the scan >
+        bpp.count(<some arguments>)  # Attempting to run a preassembled plan from an open run
+        bps.close_run()
+
+    RE(sample_plan())
+
+While it is unusual to call preassembled plans directly from within a user-defined scan, one may encounter
```

Comment:
Is this unusual? User-defined plans that glue together preassembled plans into longer routines seem common.

Reply:
I guess my wording is incorrect. Preassembled plans are typically glued together in sequential manner: once one is finished, then the other one is started. They are not put one inside of another. Should I call 'user-defined scan' as 'user-defined run'?

Reply:
Your new phrasing here seems clearer to me. :+1: